contrib/osc.complete

```diff
@@ -58,6 +58,22 @@ let count--
 test "${COMMAND_LINE:$last}" = " " && let count++
 unset last
 
+XDG_CACHE_HOME="${XDG_CACHE_HOME:-$HOME/.cache}"
+XDG_CONFIG_HOME="${XDG_CONFIG_HOME:-$HOME/.config}"
+
+for xdg_dir in "${XDG_CACHE_HOME}" "${XDG_CONFIG_HOME}"; do
+    if [ ! -d "${xdg_dir}" ]; then
+        mkdir -p "${xdg_dir}"
+    fi
+done
+
+projects="${XDG_CACHE_HOME}/osc.projects"
```

Comment:
Would it make more sense to use a folder here and save the separate cache per alias in that osc folder in `$XDG_CACHE_HOME`?

Reply:
Yes, that makes sense. Let me change it.

Reply:
OK I will match the zsh competition to do the same.